dattri/task.py

```diff
@@ -329,9 +345,7 @@ def get_param(
             layer_name (Optional[Union[str, List[str]]]): The name of the layer to be
                 extracted. If None, all the parameters will be returned. This should be
                 a string or a list of strings if multiple layers are needed. The name
-                of layer should follow the key of model.named_parameters(). If not None,
-                the layer_split will be forced to be True because a whole flattened
-                parameter will miss the layer information.
+                of layer should follow the key of model.named_parameters().
```

Comment:
Elaborate on the logical relationship between layer_name and layer_split?

Also there is some grammatical issue in the docstring for param_layer_map.